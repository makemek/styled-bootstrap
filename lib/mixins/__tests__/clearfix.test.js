import { shallow } from 'enzyme'
import { clearfix } from '../clearfix'
import styled from 'styled-components'
import React from 'react'
import 'jest-styled-components'

describe('mixin: clearfix', () => {
  it('should match snapshot', () => {
    const Comp = styled.div`
      ${ clearfix() }
    `
    const wrapper = shallow(<Comp />)
    expect(wrapper).toMatchSnapshot()
  })
})
