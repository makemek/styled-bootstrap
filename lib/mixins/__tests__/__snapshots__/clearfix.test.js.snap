// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mixin: clearfix should match snapshot 1`] = `
.c0:before,
.c0:after {
  content: " ";
  display: table;
}

.c0:after {
  clear: both;
}

<div
  className="c0"
/>
`;
